### PR TITLE
Fonts updates in the KEEP token dashboard

### PR DESCRIPTION
**Pull request by r-czajkowski**
Closes: #2065

This PR updates the fonts styles in the KEEP token dashboard based on the KEEP Design System.

### CODE REVIEW

File: solidity/dashboard/src/css/typography.less [hunk 1]
```diff
@@ -17,71 +17,81 @@ body {
...
 }
 
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  font-family: 'Work Sans', sans-serif;
+  font-feature-settings: 'tnum' on, 'lnum' on, 'zero' on;
```
Comment (Shadowfiend) [inline solidity/dashboard/src/css/typography.less:36]: Are these two necessary, given inheritance from `body`? Are there cases where the headers need to have the font but intervening elements have changed it?

File: solidity/dashboard/src/css/typography.less [hunk 2]
```diff
@@ -17,71 +17,81 @@ body {
...
+
 h1, .h1 {
-  font-family: 'Vitesse', sans-serif;
+  font-weight: 800;
+  font-size: 3rem;
+  line-height: 3.25rem;
+  letter-spacing: -2px;
+}
+
+.h1--alt {
```
Comment (Shadowfiend) [inline solidity/dashboard/src/css/typography.less:47]: Wondering if we have any way of expressing _why_ there's an `alt` here, i.e. name it in a more semantic way, or if this makes sense more as a class that can be mixed in separately. Would require a lot of reworking, so I don't think it's worth any time right now---but wanted to see if we could be thinking about that.

File: solidity/dashboard/src/css/typography.less [hunk 3]
```diff
@@ -140,28 +159,9 @@ label {
...
-
-  &-big {
-    font-weight: normal;
-    font-size: 1.125rem !important;
-    line-height: 145%;
-    color: @color-grey-60;
-    font-feature-settings: 'tnum' on, 'lnum' on;
-  }
-
   &-samll {
```
Comment (Shadowfiend) [inline solidity/dashboard/src/css/typography.less:162]: Looks like a misspelling that's probably percolated through everything :grimacing:

File: solidity/dashboard/src/css/typography.less [hunk 4]
```diff
@@ -222,20 +222,22 @@ label {
...
-    font-size: 0.7rem;
-    line-height: 1.375rem;
-    letter-spacing: 0.08rem;
+    font-weight: 500;
+    font-size: 0.5rem;
+    line-height: 0.75rem;
+    letter-spacing: 0.05rem;
   }
 
   &-intstruction {
```
Comment (Shadowfiend) [inline solidity/dashboard/src/css/typography.less:231]: Same re: misspelling/typo.

**Review by Shadowfiend — APPROVED** [review]
Approving here as everything looks good overall. Couple of things to flag, however:

- Font sizes in the right zone here look a bit small, both for buttons and text: Screen Shot 2020-10-26 at 09 14 42 IIRC this is undergoing some rework still, so that's probably fine.
- These bold 0s look very aggressive lol: Screen Shot 2020-10-26 at 09 15 31 The bold zeros in general feel a little in-your-face. /cc @liz-shinn 
- Applications / Keep Random Beacon navigation is misspelled (“Keep Rando Beacon”).
- Mouse movement between the web3 address and the dropdown can start the dropdown fading, which is a bit jarring. Couple of ways to manage this, including waiting to see if the mouse enters the dropdown before hiding it, but the easiest way would be to find some way to extend the element that does mouse tracking right up to the edge of the dropdown.
- The web3 dropdown has a pointer even when it's over things that aren't buttons—in particular, the amount of KEEP.

Let's roll this on in and keep trucking!

File: solidity/dashboard/src/css/footer.less [hunk 5]
```diff
@@ -12,15 +12,16 @@ footer.footer {
         margin-bottom: 1rem;
         
         &__item {
+            line-height: 142%;
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/css/footer.less:15]: I got this value from Figma. In Figma `line-height` for this element is 17px or 142%.

[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/css/footer.less:15]: w/o `line-height` there is too much space between items.

**Comment by liz-shinn** [comment]
> * These bold 0s look very aggressive lol: Screen Shot 2020-10-26 at 09 15 31 The bold zeros in general feel a little in-your-face. /cc @liz-shinn

Think it will look more balanced when the unit is there as well, for example:
Screen Shot 2020-10-26 at 10 38 23 AM

Also, the addresses shouldn't be in all caps case since we ran into issue with case sensitivity with addresses

**Comment by r-czajkowski** [comment]
I'll resolve this all issue in followup work, because almost all screens have changed so I'll go through page by page and update. In this PR, I focused on font updates on the css side.

File: solidity/dashboard/src/css/typography.less (hunk 3 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:162]: heh 🤦  in UI components we use `text-small` 😄. It's probably the only place with this misspelling, I'll update.

File: solidity/dashboard/src/css/typography.less (hunk 4 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:231]: same as above

File: solidity/dashboard/src/css/typography.less (hunk 2 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:47]: Hmm based on the KEEP Design System `alt(alternative)` makes sense only with h1 and h2 tag. Out of idea which tag would fit better here 🤔

File: solidity/dashboard/src/css/buttons.less [hunk 6]
```diff
@@ -32,27 +32,30 @@ button, .btn {
 
 .btn {
   &-lg {
-    padding: 1rem 2.5rem !important;
-    font-size: 1.125rem !important;
+    padding: 1rem 2.25rem !important;
+    font-size: 1.25rem !important;
+    line-height: 1.5rem !important
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/css/buttons.less:37]: I'll take a look closer at it when I have a chance. Thanks for pointing this out 👍

File: solidity/dashboard/src/css/typography.less (hunk 3 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:162]: pls see 092322f

File: solidity/dashboard/src/css/typography.less (hunk 4 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:231]: pls see 092322f

Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:231]: pls see 092322f

File: solidity/dashboard/src/css/typography.less (hunk 1 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/css/typography.less:36]: pls see c3d1712

File: solidity/dashboard/src/css/footer.less [hunk 7]
```diff
@@ -31,6 +32,6 @@ footer.footer {
     }
 
     .footer__app-version {
-        .footer-text-styles(0.5rem, @color-grey-40, 1.5rem);
+        .footer-text-styles(0.5rem, @color-grey-40, 1rem);
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/css/footer.less:35]: pls see c681076

**Pull request merged**